cmd/config_test.go

```diff
@@ -149,16 +183,17 @@ func TestConfigSet(t *testing.T) {
 	var b bytes.Buffer
 	cmdr := DeisCmd{WOut: &b, ConfigFile: cf}
 
-	err = cmdr.ConfigSet("foo", []string{"TRUE=false"})
+	err = cmdr.ConfigSet("foo", []string{"TRUE=false", "SSH_KEY=-----BEGIN OPENSSH PRIVATE KEY-----"})
```

Comment:
Yeah, it just a little confusing looking over, since you're using a private key header and base64 encoding, a combo that doesn't go together in normal usage.